Test/Optimizer/test_sr_itersolve.py

```diff
@@ -65,6 +65,8 @@ def test_qgt_solve(qgt, vstate, solver, _mpi_size, _mpi_rank):
     S = qgt(vstate)
     x, _ = S.solve(solver, vstate.parameters)
 
+    jax.tree_multimap(lambda a, b: np.testing.assert_allclose(a, b), S @ x, vstate.parameters)
```

Comment:
So I added this extra assert, which made every solver fail. I think this should work though, what's going on?